rpm/mir.spec

```
@@ -0,0 +1,630 @@
+
+%description internal-devel
+This package provides extra development files to create compositors
+built on Mir that need access to private internal interfaces.
+
+%package common-libs
+Summary:       Common libraries for Mir
+License:       LGPL-2.1-only or LGPL-3.0-only
+# mirclient is gone...
+Obsoletes:     %{name}-client-libs < 2.6.0
+# debug extension for mirclient is gone...
+Obsoletes:     %{name}-client-libs-debugext < 1.6.0
+# mir utils are gone...
+Obsoletes:     %{name}-utils < 2.0.0
+# Ensure older mirclient doesn't mix in
+Conflicts:     %{name}-client-libs < 2.6.0
+
+%description common-libs
+This package provides the libraries common to be used
+by Mir clients or Mir servers.
+
+%package lomiri-libs
+Summary:       Lomiri compatibility libraries for Mir
+License:       GPL-2.0-only or GPL-3.0-only
+Requires:      %{name}-common-libs%{?_isa} = %{?epoch:%{epoch}:}%{version}-%{release}
+Requires:      %{name}-server-libs%{?_isa} = %{?epoch:%{epoch}:}%{version}-%{release}
+
+%description lomiri-libs
+This package provides the libraries for Lomiri to use Mir
+as a Wayland compositor.
+
+%package server-libs
+Summary:       Server libraries for Mir
+License:       GPL-2.0-only or GPL-3.0-only
+Requires:      %{name}-common-libs%{?_isa} = %{?epoch:%{epoch}:}%{version}-%{release}
+
+%description server-libs
+This package provides the libraries for applications
+that use the Mir server.
+
+%package test-tools
+Summary:       Testing tools for Mir
+License:       GPL-2.0-only or GPL-3.0-only
+Requires:      %{name}-server-libs%{?_isa} = %{?epoch:%{epoch}:}%{version}-%{release}
+Recommends:    %{name}-demos
+Recommends:    glmark2
+Recommends:    xorg-x11-server-Xwayland
+Requires:      wlcs
+# mir-perf-framework is no more...
+Obsoletes:     python3-mir-perf-framework < 2.6.0
+# Ensure mir-perf-framework is not installed
+Conflicts:     python3-mir-perf-framework < 2.6.0
+
+%description test-tools
+This package provides tools for testing Mir.
+
+%package demos
+Summary:       Demonstration applications using Mir
+License:       GPL-2.0-only or GPL-3.0-only
+Requires:      %{name}-server-libs%{?_isa} = %{?epoch:%{epoch}:}%{version}-%{release}
+Requires:      inotify-tools
+Requires:      hicolor-icon-theme
+Requires:      xorg-x11-server-Xwayland
+Requires:      xkeyboard-config
+# For some of the demos
+Requires:      gnu-free-sans-fonts
+
+%description demos
+This package provides applications for demonstrating
+the capabilities of the Mir display server.
+
+%package test-libs-static
+Summary:       Testing framework library for Mir
+License:       GPL-2.0-only or GPL-3.0-only
+Requires:      %{name}-devel%{?_isa} = %{?epoch:%{epoch}:}%{version}-%{release}
+
+%description test-libs-static
+This package provides the static library for building
+Mir unit and integration tests.
+
+
+%prep
+%autosetup
+
+
+%conf
+%cmake	-GNinja %{?with_lto:-DMIR_LINK_TIME_OPTIMIZATION=ON} \
```

This is gone, we're now simply respecting flags
```suggestion
%cmake	-GNinja \
```